desktop/extensions-sdk/guides/oauth2-flow.md

```diff
@@ -51,14 +52,14 @@ window.ddClient.openExternal("https://authorization-server.com/authorize?
   &state=kH_0FdAtjCfYjOkF);
 ```
 
-## Step Two: Get the authorization code and access token
+## Step two: get the authorization code and access token
 
 ### From the extension UI
 
 > **Passing the access token**
 >
 > Currently, passing the authorization code as a query parameter to the `docker-desktop://dashboard/open` URL is not supported.
-{: .important}
+> {: .important}
```

Comment:
Probably VSCode has auto-formatted this. You need to remove it otherwise it won't be displayed correclty.

Reply:
changed back, thx